lib/msal-browser/rollup.config.js

```diff
@@ -40,21 +40,14 @@ export default [
         ]
     },
     {
-        input: "src/index.ts",
+        input: ["src/index.ts", "../msal-common/src/index.ts"],
         preserveModules: false,
         output: [
             {
                 dir: "lib",
                 format: "cjs",
                 banner: fileHeader,
                 sourcemap: true,
-            },
-            {
-                dir: "lib",
```

Comment:
Removing temporarily while I figure out how to avoid the error:

```
[!] RollupError: Invalid value "umd" for option "output.format" - UMD and IIFE output formats are not supported for code-splitting builds.
https://rollupjs.org/configuration-options/#output-format
```